docs/en/advanced-concepts/settings.md

```diff
@@ -15,6 +15,8 @@ Another example of a use-case for settings would be telling appium to ignore ele
 
 Settings are implemented via the following API endpoints:
 
+### [Update Device Settings](http://appium.io/docs/en/commands/session/settings/update-settings/)
```

Comment:
I don't think these should include the domain. Generally we've got `/docs/en/commands/session/settings/update-settings/` as the link, so that it works in Github as well as appium.io

Reply:
Great 👍
I checked some commands and `.md` also converts to `index.html` in appium.io. nice...

Reply:
Yes. that all gets done in https://github.com/appium/appium.io/blob/gh-pages/scripts/links.js